labs/README.md

```diff
@@ -0,0 +1,48 @@
+# Golden Path - Creating and deleting Flux config for an application
+
+[Golden Path tutorial - Java](https://backstage.platform.hmcts.net/catalog/default/component/golden-path-java) - VPN needed.
+
+## Creating the flux config for your lab application 
+
+Inside the labs directory run the [`create-lab-flux-config.sh`](./create-lab-flux-config.sh) script to create the flux config needed for your lab application. Use the component name you used when creating your app with backstage, which should be your GitHub username and specify the type of application you are deploying.
```

Comment:
scripts should never depend on your current working directory, please adding something like https://stackoverflow.com/questions/3349105/how-can-i-set-the-current-working-directory-to-the-directory-of-the-script-in-ba

Reply:
not pushed? scripts should be linked from the root of the repo, so probably:

```
./labs/create-lab-flux-config.sh
```

Reply:
ah sorry thought you meant something else, updated 👍🏼